packages/application-config/src/validations.ts

```diff
@@ -70,3 +70,22 @@ export const validateSubmenuLinks = (
     uriPathSet.add(uriPath);
   });
 };
+
+export const validateAdditionalOAuthScopes = (
+  config: JSONSchemaForCustomApplicationConfigurationFiles
+) => {
+  const additionalPermissionNames = new Set();
+  config.additionalOAuthScopes?.forEach(({ name }) => {
+    if (additionalPermissionNames.has(name)) {
+      throw new Error(
+        'Duplicate additional permission. Every additional permission must have a unique name'
```

Comment:
nit: could we maybe include the permission name in the error message to make it easier to developers know about the problem?
```suggestion
        'Duplicate additional permission "${name}". Every additional permission must have a unique name'
```

Reply:
Updated [5f45a61](https://github.com/commercetools/merchant-center-application-kit/pull/2822/commits/5f45a61c753574fbd2509f67a18222edaf3b0670), Thanks.